packages/compass-e2e-tests/helpers/commands/index.js

```diff
@@ -1,12 +1,7 @@
-// @ts-check
-
-/**
- * @param {ExtendedApplication} app
- */
-exports.addCommands = function (app) {
+exports.addCommands = function (compass) {
   const add = (name, path) => {
-    const f = require(path)(app);
-    app.client.addCommand(name, f);
+    const f = require(path)(compass);
+    compass.browser.addCommand(name, f);
   };
 
   add('existsEventually', './exists-eventually');
```

Comment:
Small thing, but would be nice if actual requires were used here, so that "open in tab" on click would work correctly (and I'm guessing that typing this in the follow-up PR would also be easier that way)

```suggestion
  add('existsEventually', require('./exists-eventually'));
```

Reply:
I have to change this completely in the typescript move.